### PR TITLE
Integration with Mcafee DAM

### PR DESCRIPTION
**Pull request by aviadl**
Integration with Mcafee DAM
@gshriki Please review text
@meirwah Please do the code review

### CODE REVIEW

**Comment by aviadl** [comment]
BTW it is still missing the icon

**Comment by slavikm** [comment]
**WORLDS COLLIDE**

File: Integrations/integration-mcafeeDam.yml [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    deprecated: false
+    arguments:
+    - name: id
+      required: true
+      default: true
+      description: The alert ID
+    description: Get a DAM alert from Mcafee Database Activity Monitoring by alert
+      id
+  - name: dam-get-latest-by-rule
+    deprecated: false
```
Comment (meirwah) [inline Integrations/integration-mcafeeDam.yml]: deprecated: false What is that?

File: Integrations/integration-mcafeeDam.yml [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+      default: true
+      description: Name of rule that triggered the alert
+    - name: count
+      description: Number of alerts to retrieve, defaults to 10
+    - name: timeBack
+      description: Filter DAM alerts and bring alerts that where created only in the
+        last timeBack minute, defaults to last 10 minutes
+    description: Get latest DAM alerts by rule name
+  isfetch: true
+hidden: false
```
Comment (meirwah) [inline Integrations/integration-mcafeeDam.yml]: no need for hidden, 
You need to add system: true

**Comment by meirwah** [comment]
@aviadl to merge this?

Also can you provide examples ? how to use this integration?

**Comment by aviadl** [comment]
@gshriki wants to do some changes there on the text, so not yet please

**Comment by aviadl** [comment]
get alerts by rule name:
!dam-get-latest-by-rule ruleName="New Session"

get alerts by alert id
!dam-get-alert-by-id id=34524173000

both will return a JSOn with all alert details.

If set to fetch incident, it will check fetch according to rule name, and place all attributes on the labels

**Comment by gshriki** [comment]
@aviadl - already done with the changes to the text...

**Comment by aviadl** [comment]
@meirwah please review again, and merge
Thanks

File: Integrations/integration-mcafeeDam.yml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 commonfields:
   id: McafeeDAM
   version: -1
-name: McafeeDAM
-display: McafeeDAM
+name: McAfeeDAM
```
Comment (meirwah) [inline Integrations/integration-mcafeeDam.yml]: doesn't id need to be same like name?

**Comment by meirwah** [comment]
👍

**Pull request merged**